server/core/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.kt

```diff
@@ -694,5 +700,68 @@ class HumanPoseManager(val server: VRServer?) {
 		skeleton.setPauseTracking(pauseTracking, sourceName)
 	}
 
-	fun togglePauseTracking(sourceName: String?): Boolean = skeleton.togglePauseTracking(sourceName) // #endregion
+	fun togglePauseTracking(sourceName: String?): Boolean = skeleton.togglePauseTracking(sourceName)
+
+	// This should be executed when the head tracker is changed
+	fun checkTrackersRequiringReset() {
+		// Checks if this is main human pose manager (having server) or
+		// skeleton doesn't have a head tracker or not an HMD one
+		// TODO: Should this be ran anyways if headTracker is null or is not an HMD?
+		if (server == null ||
+			skeleton.headTracker == null ||
+			skeleton.headTracker?.isHmd != true
```

Comment:
Should probably check for isComputed instead
```suggestion
			skeleton.headTracker?.isComputed != true
```